src/styles/main.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calc((#{$day-size} * 7 + (6 * #{$gap})) + #{$padding} * 2);
```

Comment:
The `max-width` calculation seems to be overly complex. It's generally a good idea to avoid complex calculations in CSS when possible for readability and maintainability. Consider simplifying this calculation or breaking it down into smaller, named variables to improve clarity.